### ISSUE TITLE
Desktop notifications broken again

### ISSUE THREAD

**Issue by terra-assistance** [RocketChat/Rocket.Chat.Electron#2270]
Desktop notifications went defunct since 3.7.0 - they never popup upon receiving a message.
Tested on numerous Win10 21H2 both 3.7.0 and 3.7.1

Please note that desktop notification "Rocket chat is still running" also disappeared in 3.7.1 (haven't checked it in 3.7.0)

**Comment by jeanfbrito** [comment]
Strange, version 3.7.1 fixed the problem with the notifications that didn't work. I have tested it on Windows 10 and Windows 11 and is working. 
Can you share more about it?

**Comment by terra-assistance** [comment]
I have 50+ Win10 21H2 x64 updated from 3.5.7 to 3.7.0 and users started reporting that notifications do not work (it's an important feature for most of them). We've installed 3.7.1 and it made no difference - still no desktop popups.
I've tested it personally - upon receiving a direct message red dot apprears on rchat systray icon but no desktop notification.

**Comment by jeanfbrito** [comment]
Its very weird, we had a problem with the injection of the code on Windows that was not making work the click on notifications to open the window. This was to have been solved in 3.7.1.
For you, nothing happens? The new message notifications and the still running, nothing shows up?

**Comment by terra-assistance** [comment]
No, the notification itself won't appear on desktop, only small red dot on systray icon indicating there are some new unread mesages available

**Comment by jeanfbrito** [comment]
I will need some help to find what is going on with your users, just tested here and all my windows shows the notifications.
![image](https://user-images.githubusercontent.com/1177482/147359573-56d69398-2002-4d1d-b121-9f94c0a83a17.png)

Did you use some kind of policy on this user's machine? The "not disturb" notifications tray is not enabled?
Help me to reproduce the problem, then I will be able to fix it. Ok?

**Comment by terra-assistance** [comment]
I've found the problem - in (Windows) Settings - System - Notifications and actions Rockat Chat was off. Which is weird because nor I disabled it neither updated any group policies recently...
Sorry for the force alarm.

Can it be reset during client update operation somehow? We always perform mass update and do not uninstall previous version first.

**Comment by jeanfbrito** [comment]
I think that this is more a windows config problem than a Rocket.Chat installation. We don't touch anything like this, even I think that we cant do that on the installer.
But, I think that Microsoft introduced some "focus assist" in Windows and some apps stopped from showing the notifications. That happened to me too in my machine with Windows 11.
But I cant say how to resolve that in a System Admin level.
If you find how to handle this, can you please tell me? Could be a good thing to add to the readme file or even a blog post.

**Comment by terra-assistance** [comment]
So I've checked several workstations - this thing with disabled Rocket.Chat notifications and actions seems to be random...
I've managed to solve it through group policy (I need it to be enabled for all users):
- User configuration - Preferences - Windows settings - Registry
- New registry item
- HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Notifications\Settings\chat.rocket
- Action = Update
- Value name = Enabled
- Value type = REG_DWORD
- Value data = 1

If Enabled = 0, then notifications are disabled. If Enabled = 1 or just missing (no such key in registry), then notifications are enabled.
I've tried an option with deleting this registry key and it didn't work - the key gets deleted by group policy but notifications won't get enabled (Rocket.Chat is listed as Off in Settings - System - Notifications and actions). Although toggling notifications via manual registry key creation/deletion works just fine.
